.github/workflows/pull-request-checks.yml

```diff
@@ -2,6 +2,9 @@ name: PR checks
 
 on:
   pull_request: {}
+  push:
```

Comment:
If that's really what we want, then we should rename the file

Reply:
k. changed name to `coinilst-ci-cd` to distinguish it from ci made by datafold